src/main/java/com/google/maps/model/GeolocationResult.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2014 Google Inc. All rights reserved.
+ *
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ * ANY KIND, either express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.google.maps.model;
+
+/**
+ * A Geolocation API result.
+ *
+ * location: The user’s estimated latitude and longitude, in degrees. Contains one lat and
```

Comment:
Can you anchor the javadoc about each of these fields on the actual field, rather than in the class?